src/_imagingcms.c

```diff
@@ -213,32 +213,31 @@ cms_transform_dealloc(CmsTransformObject *self) {
 
 static cmsUInt32Number
 findLCMStype(char *PILmode) {
-    if (strcmp(PILmode, "RGB") == 0) {
-        return TYPE_RGBA_8;
-    } else if (strcmp(PILmode, "RGBA") == 0) {
-        return TYPE_RGBA_8;
-    } else if (strcmp(PILmode, "RGBX") == 0) {
+    if (
+        strcmp(PILmode, "RGB") == 0 ||
+        strcmp(PILmode, "RGBA") == 0 ||
+        strcmp(PILmode, "RGBX") == 0
+    ) {
         return TYPE_RGBA_8;
     } else if (strcmp(PILmode, "RGBA;16B") == 0) {
         return TYPE_RGBA_16;
     } else if (strcmp(PILmode, "CMYK") == 0) {
```

Comment:
More of a style issue: these all `return`, I think it reads a little cleaner to remove the `else`s:

```suggestion
    }
    if (strcmp(PILmode, "RGBA;16B") == 0) {
        return TYPE_RGBA_16;
    }
    if (strcmp(PILmode, "CMYK") == 0) {
```
etc.

Reply:
Ok, I've updated the commit.